service/controller/azureconfig/handler/workermigration/create.go

```diff
@@ -379,6 +379,20 @@ func (r *Resource) ensureMachinePoolExists(ctx context.Context, cr providerv1alp
 		return expcapiv1alpha3.MachinePool{}, microerror.Mask(err)
 	}
 
+	var bootstrapCRRef *corev1.ObjectReference
+	{
+		s := runtime.NewScheme()
+		err := corev1alpha1.AddToScheme(s)
+		if err != nil {
+			panic(fmt.Sprintf("corev1alpha1.AddToScheme: %+v", err))
+		}
```

Comment:
is there a way to get the `Scheme` from the controller-runtime `client` instance that we have?

Reply:
I haven't been able to get the scheme from the actual client, but I was able to at least share the same scheme between the 2 references.